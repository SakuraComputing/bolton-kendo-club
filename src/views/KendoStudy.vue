<template>
    <div class="kendo-study">
        <h1>Kendo Study</h1>
        <p>Need to work out what to do with all the documentation</p>
    </div>    
</template>

<style>

.kendo-study {
    margin: 8%;
}
</style>

