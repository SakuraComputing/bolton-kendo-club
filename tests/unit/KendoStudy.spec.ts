import { shallowMount } from '@vue/test-utils'
import KendoStudy from '@/views/KendoStudy.vue'

describe('KendoStudy.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(KendoStudy);
    expect(wrapper).toMatchSnapshot()
  })
})
