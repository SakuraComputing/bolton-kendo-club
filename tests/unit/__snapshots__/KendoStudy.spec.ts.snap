// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KendoStudy.vue renders correctly 1`] = `
<div class="kendo-study">
  <h1>Hanshi Says</h1>
  <p>Sumi Sensei Hanshi Hachi-dan</p> <img src="https://www.kendoguide.org/wp-content/uploads/2011/06/Masatake-Sumi-sensei.jpg" alt="Sumi Sensei">
  <p>To make ippon you need to meet the following criteria</p>
  <ul class="kendo-study-list">
    <li>1. Full Spirit</li>
    <li>2. Proper Posture</li>
    <li>3. Cut with the proper part of the shinai</li>
    <li>4. Hit proper part of armour</li>
    <li>5. Correct direction of sword edge</li>
    <li>6. Zanshin</li>
  </ul>
</div>
`;
